packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -114,6 +114,12 @@ abstract class TransitionRoute<T> extends OverlayRoute<T> {
   AnimationController get controller => _controller;
   AnimationController _controller;
 
+  /// The animation for the route being pushed on top of this route. This
+  /// animation lets this route coordinate with the entrance and exit transition
+  /// of routes pushed on top of this route.
```

Comment:
of the route being pushed on top of 

Doesn't the secondary animation also apply to the route that's being exposed because the route above it is being popped?

Reply:
I just moved this block from below but all these docs are wrong. Re-editing them.

Reply:
It's not "routes pushed on top of this route" plural, it's just the one route pushed on top of this one?

Reply:
Thanks, indeed it should be singular